tools/cbmc/proofs/MakefileCommon.json

```diff
@@ -74,7 +74,9 @@
         "--32",
         "--bounds-check",
         "--pointer-check",
-        "--pointer-primitive-check"
+        "--pointer-primitive-check",
+	"--malloc-fail-null",
+	"--malloc-may-fail"
```

Comment:
The indentation here looks odd ... I suspect there's a mix of tabs and spaces in the two lines above?